README.md

```diff
@@ -1,8 +1,5 @@
 # Getting Started with the .NET MAUI SignaturePad 
-A quick-start project that helps you integrate the Signature Pad control with the .NET MAUI application by adding the Syncfusion.Maui.SignaturePad package from NuGet.
-
-Refer to the following documentation for the Syncfusion .NET MAUI SignaturePad Control: 
-https://help.syncfusion.com/maui/signaturepad/getting-started
+The [.NET MAUI SignaturePad](https://www.syncfusion.com/maui-controls/maui-signaturepad?utm_source=github&utm_medium=listing&utm_campaign=maui-signaturepad-github-samples) is an interactive UI control that allows users to capture smooth and realistic signatures. Also, users can save their signatures as images and sync them across devices and documents that need signatures. A quick-start project that helps you integrate the Signature Pad control with the .NET MAUI application by adding the Syncfusion.Maui.SignaturePad package from NuGet.

```

Comment:
No need for this line
![image](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-signature-pad/assets/95272441/9fd5a842-331a-4632-a2ab-9109cdb212c4)

Reply:
resolved